Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp

```diff
@@ -619,6 +636,28 @@ static void webkit_web_context_class_init(WebKitWebContextClass* webContextClass
             nullptr,
             static_cast<GParamFlags>(WEBKIT_PARAM_READWRITE | G_PARAM_CONSTRUCT_ONLY));
 
+    /**
+     * WebKitWebContext:application-id:
+     *
+     * The reverse DNS application ID for this web context. Setting this property is specially
+     * required when the Application creating WebViews is not spawned using #GApplication nor
+     * Flatpak. In this scenario the internal WebKit process launcher spawns its sub-processes in a
+     * Bubblewrap sandbox that requires some form of application identifier.
```

Comment:
This is true only in the new API version, so... maybe expose this API only for new API version? Well, maybe it's OK to expose it unconditionally to ease migration.

Reply:
I don't think a property of the web context is the right way. The same application can have more than once web context, but createFlatpakInfo caches the data, so the first context id is the one used and the others ignored. I guess it doesn't make sense to use a different app id in every context, but this API allows that. We could probably use a different flatpak info per web process, but still it doesn't look like it's useful.